sp_gp:
    kernel_type: erl::covariance::Matern32<float, 2>
    kernel_setting_type: erl::covariance::Covariance<float>::Setting
    kernel:
        x_dim: 2
        scale: 0.18
        scale_mix: 1
        weights: [ ]
    max_num_samples: 2000
    sparse_zero_threshold: 1e-6
    use_sparse: false
    diagonal_qm: false
max_distance: 30
free_points_per_meter: 3
free_sampling_margin: 0.01
parallel: true
logodd_free: -1
logodd_occupied: 1
logodd_variance: 1.0e-4
